### ISSUE TITLE
make false positive debugging easier

### ISSUE THREAD

**Issue by Quasilyte** [go-critic/go-critic#476]
We may want a way to display context lines for warnings.
For example, instead of this:
```
$GOROOT/src/image/gif/writer_test.go:325:10: elseif: can replace 'else {if cond {}}' with 'else if cond {}'
```
I would like to see:
```
$GOROOT/src/image/gif/writer_test.go:325:10: elseif: can replace 'else {if cond {}}' with 'else if cond {}'
		if upperBound >= 8 {
			if err != nil {
				t.Errorf("upperBound=%d: %v", upperBound, err)
			}
		} else {
			if err == nil {
				t.Errorf("upperBound=%d: got nil error, want non-nil", upperBound)
			}
		}
```
This is not intended to be a default reporting mode.
Just a debug (verbose) mode that makes it simpler for investigating potential false positives.

Or maybe I need to run gocritic under Emacs and jump to warning location in 1 click..
I'll think about that.

**Comment by cristaloleg** [comment]
I prefer to just clicking on a filepath. So the output of go-critic will not be bloated with if's/switches or smth else.
From the other side we've a task for adding `-verbose` but I don't know why do we need this?

**Comment by Quasilyte** [comment]
OK, let's close this one in favor of `-verbose` proposal.
